Ps 117 centreon ha 4nodes updates (#1984)
centreon/centreon-documentation: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md

In order to adhere to best practices and be as resilient as possible, the Quorum server placement
Quorum server should be at a different site than the two primary nodes, with independent network attachments.

> > **WARNING:** Be sure Selinux and Firewalld are disabled.

Reviewer: ```suggestion
> **WARNING:** Be sure SELinux and Firewalld are disabled.
```

    password: "@MARIADB_CENTREON_PASSWD@"
```

> And then you need to restart gorgone and cbd_central_broker resources to changes take effects.

Reviewer: ```suggestion
And then you need to restart the gorgone and cbd_central_broker resources for changes to take effect.
```

> Restarting gorgone resource will restart all resources below, so using the following command:

Reviewer: ```suggestion
Use the following command to restart the gorgone resource and all subsequent resources:
```

```bash
pcs resource restart gorgone
```

> And after resources are restarted, verify if all is OK with the command `crm_mon -fr`, the result should be like that:

Reviewer: ```suggestion
After resources are restarted, verify if all is OK using the command `crm_mon -fr`. The result should look like this:
```

    * Stopped: [ @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ ]
  * vip_mysql   (ocf::heartbeat:IPaddr2):        Started centreon-rhel8-pri-bdd
  * Clone Set: php-clone [php]:
    * Started: [ @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ ]
    * Stopped: [ centreon-rhel8-pri-bdd @DATABASE_SLAVE_NAME@ ]
  * Clone Set: cbd_rrd-clone [cbd_rrd]:
    * Started: [ @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ ]
    * Stopped: [ centreon-rhel8-pri-bdd @DATABASE_SLAVE_NAME@ ]
  * Resource Group: centreon:
    * vip       (ocf::heartbeat:IPaddr2):        Started @CENTRAL_MASTER_NAME@
    * http      (systemd:httpd):         Started @CENTRAL_MASTER_NAME@
    * gorgone   (systemd:gorgoned):      Started @CENTRAL_MASTER_NAME@
    * centreon_central_sync     (systemd:centreon-central-sync):         Started @CENTRAL_MASTER_NAME@
    * cbd_central_broker        (systemd:cbd-sql):       Started @CENTRAL_MASTER_NAME@
    * centengine        (systemd:centengine):    Started @CENTRAL_MASTER_NAME@
    * centreontrapd     (systemd:centreontrapd):         Started@CENTRAL_MASTER_NAME@
    * snmptrapd (systemd:snmptrapd):     Started @CENTRAL_MASTER_NAME@

Migration Summary:
```

> If an error is found while your resources are running, try "cleaning" with this command:

Reviewer: ```suggestion
If an error is found while your resources are running, try doing some "cleaning" using this command:
```
